catalyst_voices/apps/voices/lib/widgets/text_field/voices_text_field.dart

```diff
@@ -98,6 +98,9 @@ class VoicesTextField extends StatefulWidget {
   /// [MaxLengthEnforcement]
   final MaxLengthEnforcement? maxLengthEnforcement;
 
+  /// [MouseCursor]
```

Comment:
```suggestion
  /// [TextField.mouseCursor]
```

Can you please update above comments too? For `MaxLengthEnforcement` and `AutovalidateMode`

Reply:
Done :D